usr/lib/zabbix-check/bin/zabbix-check-sshkey

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env sh
+
+authorized_keyfile=${1}
+
+
+fail() {
+    echo "ZBX_NOTSUPPORTED"
+    exit 1
+}
+
+
+[ -r "${authorized_keyfile}" ] || fail
+
+md5sum "${authorized_keyfile}" | cut -d" " -f1
```

Comment:
We should consider switching to sha512sum.

Reply:
done